### PR TITLE
Update docs to use “SQT” instead of “QUOTE” 

### PR DESCRIPTION
**Pull request by fsargent**
Default keymap used an invalid character, this sets it valid.

### CODE REVIEW

**Review by petejohanson — APPROVED** [review]
Thanks!

File: docs/docs/keymap-example.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 //                               |   GUI   |   DEL   | RETURN  |  SPACE  | ESCAPE  |  |  RETURN |  SPACE  |   TAB   |   BSPC  |  R ALT  |
 			bindings = <
     &kp ESC    &kp Q    &kp W    &kp E     &kp R     &kp T                                                 &kp Y     &kp U     &kp I     &kp O     &kp P    &kp BSLH
-    &kp TAB    &kp A    &kp S    &kp D     &kp F     &kp G                                                 &kp H     &kp J     &kp K     &kp L     &kp SEMI &kp QUOTE
+    &kp TAB    &kp A    &kp S    &kp D     &kp F     &kp G                                                 &kp H     &kp J     &kp K     &kp L     &kp SEMI &kp RQT
```
Comment (petejohanson) [inline docs/docs/keymap-example.md:13]: Actually, typo here:

```suggestion
    &kp TAB    &kp A    &kp S    &kp D     &kp F     &kp G                                                 &kp H     &kp J     &kp K     &kp L     &kp SEMI &kp SQT
```

**Review by petejohanson — CHANGES_REQUESTED** [review]

File: docs/docs/keymap-example.md (hunk 1 above)
Comment (innovaker) [inline docs/docs/keymap-example.md:13]:
```suggestion
    &kp TAB    &kp A    &kp S    &kp D     &kp F     &kp G                                                 &kp H     &kp J     &kp K     &kp L     &kp SEMI &kp SQT
```
???

**Comment by petejohanson** [comment]
@innovaker https://tenor.com/XE21.gif

**Review by Nicell — APPROVED** [review]

**Review by petejohanson — APPROVED** [review]

**Pull request merged**